packages/grafana-ui/src/components/VizLegend/VizLegend.tsx

```diff
@@ -27,6 +28,7 @@ export const VizLegend: React.FunctionComponent<LegendProps> = ({
           placement={placement}
           sortBy={sortKey}
           sortDesc={sortDesc}
+          eventBus={eventBus}
```

Comment:
Same comment as in the PieChart - I think grafana/ui components don't need knowledge about events bus existence